x-pack/plugin/search-business-rules/src/main/java/org/elasticsearch/xpack/searchbusinessrules/SearchBusinessRulesFeatures.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.searchbusinessrules;
+
+import org.elasticsearch.features.FeatureSpecification;
+import org.elasticsearch.features.NodeFeature;
+
+import java.util.Set;
+
+public class SearchBusinessRulesFeatures implements FeatureSpecification {
+
+    public static final NodeFeature PINNED_RETRIEVER_FEATURE = new NodeFeature("pinned_retriever");
```

Comment:
Nitpick: Typically we define `NodeFeature`s in the class that implements them, so for this would be the `PinnedRetrieverBuilder`.